homeassistant/components/point/binary_sensor.py

```diff
@@ -40,6 +40,7 @@
         "humidity_high",
         "humidity_dropped_normal",
     ),
+    "motion": ("pir_motion", ""),  # pir_motion when motion detected
```

Comment:
The details should be in the pypoint lib.

The format of the `EVENTS` dict is not ideal, and the integration will use unsupported device classes.
Instead of making a copy of what's in the lib, we should import `EVENTS` from the lib and in the integration have a dict of what we want so use from the lib, in which each item defines either a proper device class (when HA defines one) or an icon (when it doesn't).
e.g.
```py
BINARY_SENSORS = {
    "battery": {"device_class": DEVICE_CLASS_BATTERY},
    "button_press": {"icon": "mdi:xyz"},
}
```

Reply:
I'm on it..